docs/template/public/less/_kss-content.less

```diff
@@ -0,0 +1,206 @@
+// shared styles across pages 
+#kss-content {
+    .sectionSpacing {
+        padding-bottom: 30px;
+        padding-top: 30px;
+    }
+    .table-behavoir-cell .seperator {
+        margin: 0 50px 0 0;
+    }
+
+    /* patterns */
+    .row.visual-code {
+        margin: 0;
+    }
+
+    h3.popover-title {
+        margin-bottom: 0;
+    }
+
+    .usercard-info p {
+        margin-bottom: 0;
+    }
+
+    .list-group-item.deleteable .list-group-item-delete {
+        top: 18%;
+    }
+
+    .row {
+        margin-right: -8px;
+        margin-left: -8px;
+    }
+
+    .modal-dialog,.popover {
+        z-index: 0;
+    }
+
+    .circle {
+        border: 1px solid #cfd8e3;
+        box-shadow: 0px 1px 1px #fff;
+        margin: 0 auto 20px auto;
+        width: 100%;
+        height: 0;
+        padding: 50% 0;
+        border-radius: 50%;
+        
+    }
+
+    .circle-type {
+        font-size: 74px;
+        display: inline-block;
+        margin-top: -35%;
+    }
+
+    .seperator {
+        margin: 40px 0 80px 0;
+        position: relative;
+
+        .hexagon {
+            top: 31px;
+            left: 0;
+            position: relative;
+            width: 36px;
+            height: 20.78px;
+            background-color: #d8dfe8;
+            margin: auto;
+            display: block;
+            z-index: 2;
+
```

Comment:
done. 

remove all hexagon code as it is no longer needed. 

retained 3 levels for circle code because there is a generic circle and then the sizing and alignment for the circles that go with separator for the code snippet expand/collapse.